licenses/LICENSE_mpl-2

```diff
@@ -1,373 +0,0 @@
-Mozilla Public License Version 2.0
```

Comment:
this file had to be added explicitly as a copy of the licence had to be provided, and it's not added in the build-time version.

Reply:
This license was there for some reason manually for OkHttp, but we don't use it anywhere. For that reason we don't need that license. If we at any time add OkHttp as dependency, it would be added back automatically.